cpp/CMakeLists.txt

```diff
@@ -321,15 +321,30 @@ endif(PER_THREAD_DEFAULT_STREAM)
 
 if(BUILD_TESTS)
     include(CTest)
-    include(ConfigureGoogleTest)
+    find_package(GTest)
+    
+    if(GTEST_FOUND)
+        set(GTEST_INCLUDE_DIR "$ENV{CONDA_PREFIX}/include/gtest")
```

Comment:
We can't assume that gtest is found in a conda environment, could have been installed via apt/yum/spack/etc.